### DISCUSSION TITLE
Add ability to save workspaces

### DISCUSSION THREAD

**Discussion by lukamrkonjic** [Floorp-Projects/Floorp#1445] [Feature Request]
Now, if I make three workspaces, add links to them and close the browser without first ticking the "open same windows before closure", all my workspaces are gone along with their opened tabs. I've even tried checking this checkbox but despite that, my workspaces sometimes disappear on Floorp start.

I wish we could save workspaces permanently and their opened links.

**Comment by CutterKnife** [comment]
Normally, windows and Workspaces can be restored under "Recently closed windows" in the History menu.
The ability to synchronize Workspaces will be a feature of Floorp 12.

**Comment by ledsales** [comment]
I second this request, in Vivaldi I can just save the current session, which saves all tabs and workspaces, that way if something does happen, they can just be reloaded in one click. I think not being able to save sessions is one of just two issues that will stop me using floorp as my main browser.

**Comment by theLittlePrinceAD** [comment]
Saving Workspoaces is more than synchronizing.
I work with a lot of groups of tabs (workspaces) and often close or open new tabs, and want to have the possibility to go back to a previous combination of tabs. An automatic save of all Workspaces with their tabs at a time interval and also the possibility for a manual save is necessary.
